packages/medusa/src/strategies/cart-completion.ts

```diff
@@ -228,20 +228,28 @@ class CartCompletionStrategy extends AbstractCartCompletionStrategy {
       return res
     }
 
-    const cart = await this.cartService_
-      .withTransaction(manager)
-      .authorizePayment(id, {
-        ...context,
-        cart_id: id,
-        idempotency_key: idempotencyKey,
-      })
+    const txCartService = this.cartService_.withTransaction(manager)
+
+    let cart = await txCartService.retrieve(id, {
```

Comment:
question: Why are we converting `authorizePayment` to accept a cart, if we are not going to use it?